### ISSUE TITLE
Bouton TERMINER dans les activités.

### ISSUE THREAD

**Issue by nimdanor** [PlatonOrg/platon#191]
BON cela ne vas pas les élèves sortent déffinitivement de l'activité alors qu'elle n'est pas noté et qu'ils souhaite faire les exercices...

Un il faut ajouter un choix dans la navigation qui est "fermer pour revenir" qui ne fait que fermer l'onglet !

Il faut un moyen pour les profs de ré-ouvrir l'activité pour certains élèves.
Actuellement c'est possible ave BDBeaver ?

**Comment by NewMeeh** [comment]
Oui c'est possible de réouvrir les activités "à la main" avec dbeaver ou autre accès à la base de données

**Comment by nimdanor** [comment]
Ok comment je fait sur dbbeaver ?

Sinon il faut un bouton sur les activités dans un cours pour gérer cet aspect la en effet c’est le truc qui nous est arrivé en tp noté.
Il n’y a pas d’interface direct pour les profs et c’est un problème

**Comment by NewMeeh** [comment]
Oui, nous allons rajouté une interface pour que les enseignants puissent réouvrir les activités pour les élèves. 

Pour le faire manuellement c'est assez fastidieux, il faut trouver la session de l'activité concernée, récupérer le JSON du champs `variables` puis modifier le champs `terminated` de `false` à `true`

**Comment by nimdanor** [comment]
L’interface existe ?
Mais elle n’est pas déployée ?

**Comment by NewMeeh** [comment]
L'interface n'existe pas encore. Mais ce n'est pas très long à développer, nous pourrions ajouter l'option dans l'interface des statistiques ? 

![image](https://github.com/user-attachments/assets/a8f6ef76-922a-4479-b129-942100519d69)

**Comment by nimdanor** [comment]
Ou dans un onglet spécifique. Avec un verrou comme icon ;)
